### DISCUSSION TITLE
Filter by column header in saved search (like Excel Auto-Filter)

### DISCUSSION THREAD

**Discussion by atrent0815** [burkybang/SuiteAdvanced-Feedback#61] [Ideas]
It would be convenient to allow users to sort, search, and filter saved searches by clicking on any column header and being able to dynamically sort A to Z, search by text containing, or select all rows with a matching result to a dropdown. This would supplement the existing criteria and filters by adding quick search and sort to each unique column of data. Something similar to excel auto-filters https://support.microsoft.com/en-us/office/filter-data-in-a-range-or-table-01832226-31b5-4568-8806-38c37dcc180e 
![Screenshot 2022-11-02 155244](https://user-images.githubusercontent.com/117312828/199589281-38aa88e2-6ce9-41f5-8ede-36a5f8c69767.png)

**Comment by burkybang** [comment]
Wow that's certainly a big and creative idea! I'll have to give that some thought to determine plausibility.

**Reply by felixdivall** [reply]
I came here to ask for this. Filtering records by text should really be a basic feature If this is made possible I think more will understand how useful this extension is.

**Comment by NicoleAnswer** [comment]
This is an awesome idea.
